src/symmetry.jl

```diff
@@ -243,19 +243,22 @@ Symmetrize a density by applying all the basis (by default) symmetries and formi
     G_to_r(basis, ρout_fourier ./ length(symmetries))
 end
 
-# symmetrize the stress tensor, which is a rank-2 contravariant tensor in reduced coordinates
-function symmetrize_stresses(lattice, symmetries, stresses)
+# symmetrize the stress tensor, a mtrix in cartesian coordinates
+function symmetrize_stresses(model::Model, stresses, symmetries)
+    # see (A.28) of https://arxiv.org/pdf/0906.2569.pdf
     stresses_symmetrized = zero(stresses)
     for symop in symmetries
-        S_reduced = inv(lattice) * symop.S * lattice
-        stresses_symmetrized += S_reduced' * stresses * S_reduced
+        W_red, _ = get_Ww(symop)
+        W = matrix_red_to_cart(model, W_red)
+        stresses_symmetrized += W * stresses / W
```

Comment:
I'd rather go for consistency for:
```suggestion
        W, _ = get_Ww(symop)
        W_cart = matrix_red_to_cart(model, W)
        stresses_symmetrized += W_cart * stresses / W_cart
```